### ISSUE TITLE
InsightFace-Pytorch vs. InsightFace-v2

### ISSUE THREAD

**Issue by yossibiton** [foamliu/InsightFace-v2#25]
there is another similar repository called InsightFace-PyTorch :
https://github.com/foamliu/InsightFace-PyTorch

What's the difference between the 2 repositories ? which one is recommended ?

**Comment by foamliu** [comment]
InsightFace-Pytorch uses MegaFace as validation set, while InsightFace-v2 uses LFW.
